app/forms/policy_form.rb

```diff
@@ -80,6 +81,8 @@ def update(attrs)
   end
 
   def save
+    return unless self.policy.valid? || valid?
```

Comment:
Further, not part of this PR I know but this class contains an `attr_accessor :policy`. This should only be an `attr_writer` as we override the reader behaviour later on in the class definition.